ros2/images.yaml.em

```diff
@@ -23,6 +23,8 @@ images:
             - python3-pip
             - python3-setuptools
             - python3-vcstool
+            - libtinyxml-dev
```

Comment:
Just curious, not quite up to speed on recent ros2 changes. What ros2 doing with these new packages and libs?